keyboards/flygone60/rev3/keymaps/via/keymap.c

```diff
@@ -0,0 +1,89 @@
+/* Copyright 2021 ShandonCodes
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _0,
+    _1,
+    _2,
+    _3
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    SHILL = SAFE_RANGE,
```

Comment:
```suggestion
    SHILL = USER00,
```

Do this, and you can add customkeycodes section to the via json and have it show up in via.

Reply:
Thanks for the tip, I just added the changes!!!